lib/worker/src/rpc.rs

```diff
@@ -158,14 +164,22 @@ impl<E: EVMClient + 'static> publications::Server for Publications<E> {
                 return Err(Error::failed("namespace not found".into()));
             }
 
+            let cache_duration = db::get_cache_config(&p, &ns, &rel).await?;
+
             let upload_type = UploadType::Multipart(Box::new(Object {
                 name: filename.clone(),
                 content_type: Some("application/octet-stream".into()),
                 size: size as i64,
-                metadata: Some(HashMap::from([(
-                    "timestamp".into(),
-                    format!("{}", timestamp),
-                )])),
+                metadata: Some(HashMap::from([
```

Comment:
we gotta send the cache_duration to GCS because it's the CF that is inserting the entries in the `jobs` table. See https://github.com/tablelandnetwork/basin-storage/pull/10